go.mod

```diff
@@ -3,57 +3,74 @@ module github.com/open-telemetry/opentelemetry-collector
 go 1.13
 
 require (
```

Comment:
Are these new dependencies added manually? If so why, are they needed for the PR?

Reply:
no, they were added when I ran `go get -u` to upgrade the specific broken deps mentioned in #611 - the prometheus discovery, and the k8s.io discovery.